tools/rapids-get-pr-wheel-artifact-github

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Echo path to an artifact for a specific PR from GitHub Artifacts.
+# Finds and uses the latest commit on the PR.
+#
+# Positional Arguments:
+#   1) repo name
+#   2) PR number
+#   3) "cpp" or "python", to get the artifact for the C++ or Python build, respectively
+#   4) [optional] commit hash, to get the artifact for a specific commit
+#
+# RAPIDS_PY_WHEEL_NAME must be set to indicate the package name to fetch.
+# Example Usage:
+#   rapids-get-pr-wheel-artifact-github rmm 1095 cpp
+set -euo pipefail
+
+repo="${1}"
+pr="${2}"
+package_type="${3}"
+commit="${4:-}"
+
+# Check if RAPIDS_PY_WHEEL_NAME is set when fetching a wheel
+if [[ -z "${RAPIDS_PY_WHEEL_NAME:+placeholder}" ]]; then
```

Comment:
```suggestion
if [[ -z "${RAPIDS_PY_WHEEL_NAME}" ]]; then
```

I don't think `+placeholder` is providing anything here, but let me know if I'm missing something.